course/.gitignore

```diff
@@ -0,0 +1,2 @@
+/build
+/.vscode
```

Comment:
I'm ok with this, but keep in mind that usually is not a good idea to update the .gitignore file in the clients repo to ignore files in your own personal computer setup.